### PR TITLE
Start using gomods

### LINKED ISSUE #206: Dependencies
Go started to provide solution for dependency management: `go mod` command.

Are there any plans to start managing dependencies (instead of "just" go get them) and make gobetween goget-able?

### PR DESCRIPTION
**Pull request by kron4eg**
Fixes #206

### CODE REVIEW

**Comment by kron4eg** [comment]
cc @yyyar @illarion

File: discovery/srv.go [hunk 1]
```diff
@@ -87,18 +87,18 @@ func srvFetch(cfg config.DiscoveryConfig) (*[]core.Backend, error) {
 
 			resp, err := srvDnsLookup(cfg, record.Target, dns.TypeA)
 			if err != nil {
-				log.Warn("Error fetching A record for ", record.Target, " skipping...")
+				log.Warn("Error fetching A record for ", record.Target, " skippinggithub.com/yyyar/gobetween.")
```
Comment (yyyar) [inline discovery/srv.go]: Seems like incorrect ".." replacement, should remain "skipping..." :-)

Comment (kron4eg) [reply discovery/srv.go]: oh yeah, hehehe. Fixed.

**Comment by yyyar** [comment]
Merged. @kron4eg thanks for contribution! :+1:

**Pull request merged**